data/tlg1269/tlg002/__cts__.xml

```diff
@@ -0,0 +1,7 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg1269" xml:lang="grc" urn="urn:cts:greekLit:tlg1269.tlg002">
+    <ti:title xml:lang="lat">Cleanthis Assii Fragmenta et Apophthegmata.</ti:title>
```

Comment:
I would stay with the spreadsheet on this. 
<title xml:lang="lat">Testimonia et fragmenta</title>